packages/application-components/src/components/dialogs/internals/dialog-container.tsx

```diff
@@ -63,70 +65,80 @@ const GridArea = styled.div<GridAreaProps>`
   grid-area: ${(props) => props.name};
 `;
 
-const DialogContainer = (props: Props) => (
-  <ClassNames>
-    {({ css: makeClassName }) => (
-      <Modal
-        isOpen={props.isOpen}
-        onRequestClose={props.onClose}
-        shouldCloseOnOverlayClick={Boolean(props.onClose)}
-        shouldCloseOnEsc={Boolean(props.onClose)}
-        overlayElement={getOverlayElement}
-        overlayClassName={makeClassName(getOverlayStyles(props))}
-        className={makeClassName(getModalContentStyles(props))}
-        contentLabel={props.title}
-        parentSelector={props.getParentSelector}
-        ariaHideApp={false}
-      >
-        <GridArea name="top" />
-        <GridArea name="left" />
-        <GridArea name="right" />
-        <GridArea name="bottom" />
-        <GridArea
-          name="main"
-          css={css`
-            display: flex;
-            flex-direction: column;
-            align-items: center;
-            justify-content: center;
-            height: 100%;
-            overflow: hidden;
-          `}
+const DialogContainer = (props: Props) => {
```

Comment:
Changes here are mainly because of the indentation.
The relevant part is adding the warning for the accessible title.